go.mod

```diff
@@ -1,34 +1,68 @@
 module github.com/ONSdigital/dp-table-renderer
 
-go 1.19
+go 1.21
 
 require (
 	github.com/360EntSecGroup-Skylar/excelize v1.4.1
-	github.com/ONSdigital/dp-healthcheck v1.5.0
+	github.com/ONSdigital/dp-healthcheck v1.6.0
 	github.com/ONSdigital/dp-net v1.5.0
-	github.com/ONSdigital/log.go/v2 v2.3.0
+	github.com/ONSdigital/dp-net/v2 v2.11.1
```

Comment:
It appears that a `go mod tidy` is needed as I believe this is an indirect dependency.